src/commands/cloud-run/__tests__/renderer.test.ts

```diff
@@ -1,5 +1,36 @@
+import process from 'process'
+
+import {MOCK_CWD} from '../../../helpers/__tests__/fixtures'
+import {renderAdditionalFiles, renderProjectFiles} from '../../../helpers/renderer'
+
 import {renderAuthenticationInstructions} from '../renderer'
 
-test('renderAuthenticationInstructions', () => {
-  expect(renderAuthenticationInstructions()).toMatchSnapshot()
+process.cwd = jest.fn().mockReturnValue(MOCK_CWD)
+
+describe('renderer', () => {
+  test('renderAuthenticationInstructions', () => {
+    expect(renderAuthenticationInstructions()).toMatchSnapshot()
+  })
+
```

Comment:
I don't think we should be testing this inside of the `cloud-run` folder.

Reply:
Why not? `renderAuthenticationInstructions()` lives in `cloud-run/renderer.ts`

Reply:
Oh, not that, the ones below it, idk why it shows that one.

We normally don't test renderers because your snapshots already show them.

Reply:
Makes sense, I have removed the renderer tests